UI/window-basic-settings.cpp

```diff
@@ -3072,11 +3070,13 @@ void OBSBasicSettings::SaveOutputSettings()
 	SaveSpinBox(ui->advOutFFABitrate, "AdvOut", "FFABitrate");
 	SaveEncoder(ui->advOutFFAEncoder, "AdvOut", "FFAEncoder");
 	SaveEdit(ui->advOutFFACfg, "AdvOut", "FFACustom");
-	SaveTrackIndex(main->Config(), "AdvOut", "FFAudioTrack",
-			ui->advOutFFTrack1, ui->advOutFFTrack2,
-			ui->advOutFFTrack3, ui->advOutFFTrack4,
-			ui->advOutFFTrack5, ui->advOutFFTrack6);
-
+	config_set_int(main->Config(), "AdvOut", "FFAudioTrack",
+		(ui->advOutFFTrack1->isChecked() ? (1 << 0) : 0) |
```

Comment:
Style:  Indent line continuations with two tabs.

Is there any difference between this and using `SaveTrackIndex` instead?  I'm not sure I'm a fan of ternaries here, but I guess we'll have to see how Jim feels about this syntax.

Reply:
syntax is that of Jim for standard recording, cf line 3048-3054 of same file

Reply:
`SaveTrackIndex` is still called above that at Line 3030, though.